oss-cloud-backend/serverless.yml

```diff
@@ -18,7 +18,7 @@ functions:
           method: post
 
   getAllContributors:
-    handler: handler.getAllContributors
+    handler: handler.getAllContributorsAndCount
```

Comment:
I would suggest to name the handler and lambda simply `getContributors`

Reply:
Handler and lambda renamed.